liangBarskyAlgo/liangBarskyAlgo.cpp

```diff
@@ -0,0 +1,94 @@
+#include<iostream.h>
+#include<graphics.h>
+
+void main()
+{
+    int gd = DETECT, gm;
+    /** x1, x2, y1 & y2 are line coordinates
+        xmax, xmin, ymax & ymin are window's lower and upper corner coordinates
+        xx1, yy1, xx2 & yy2 are the new coordinates of clipped line
+    **/
+    int x1, y1, x2, y2, xmax, xmin, ymax, ymin, xx1, yy1, xx2, yy2, dx, dy, i;
+    int p[4], q[4];
+    float t1, t2, t[4];
+
+    initgraph (& gd, & gm, "C:\\TC\\BGI");
+
+    // Enter the coordinates for window
+    cout<<"Enter the lower co-ordinates of window";
```

Comment:
Add space around `<<`, `>>` etc.

Reply:
@jainaman224 Actually I told him to remove space around << and >> in cin & cout statements :sweat_smile: 
As it is not mentioned in coding conventions and the compiler simply ignores them soo...

Reply:
But it is good practice bro. @robotjellyzone Please make other changes. We will fix these.